data-esnext.js

```diff
@@ -778,6 +778,7 @@ exports.tests = [
     babel: true,
     jsx: true,
     chrome58: 'flagged',
+    typescript: typescript.corejs,
```

Comment:
Thanks, that clarifies.

However, that transpiled code violates the spec; it doesn't check for enumerability of symbols.